docs/en/guides/20-cloud/10-using-databend-cloud/06-aws-iam-role.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Access your own AWS S3 bucket in Databend Cloud with IAM Role"
+---
+
+# Why IAM Role
+
+With AWS IAM Role, you can access your own AWS S3 buckets in Databend Cloud. This allows you to securely access your data and perform data analysis without having to manage your AWS credentials.
+
+# How to Use IAM Role
+
+1. Get the IAM role ARN for your Databend Cloud organization(ask for support):
```

Comment:
how about "Raise a support ticket to tet the IAM role ARN for your Databend Cloud organization"